### PR TITLE
Fixed some trivial formatting issues in test/material/popup_menu_test.dart

### PR DESCRIPTION
**Pull request by HansMuller**
Noticed some lines from https://github.com/flutter/flutter/pull/81686 with 2-space instead of 4-space indent after #81686 had landed.

### CODE REVIEW

**Review by Piinks — APPROVED** [review]
LGTM once @Hixie approves test-exemption. :)

File: packages/flutter/test/material/popup_menu_test.dart
Comment (darrenaustin) [inline packages/flutter/test/material/popup_menu_test.dart:306]: Is this a personal preference for += 1 instead of ++ or is this something spec'd in the style guide?

**Review by darrenaustin — APPROVED** [review]
LGTM

File: packages/flutter/test/material/popup_menu_test.dart
Comment (HansMuller) [reply packages/flutter/test/material/popup_menu_test.dart:306]: Ian asked for += 1 instead of ++ in a review ages ago, so I've developed that habit.

Comment (Hixie) [reply packages/flutter/test/material/popup_menu_test.dart:306]: see the style guide :-)

**Comment by Hixie** [comment]
test-exempt: code formatting

File: packages/flutter/test/material/popup_menu_test.dart
Comment (HansMuller) [reply packages/flutter/test/material/popup_menu_test.dart:306]: FTR: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#prefer-over

Comment (darrenaustin) [reply packages/flutter/test/material/popup_menu_test.dart:306]: Yup thx. Sorry for not just looking it up in the first place 😮. Also not sure why it didn't sink in for me given that I have read through that twice before. Must be too many style guides in my past starting to blend together.

**Pull request merged**
